.github/workflows/create_release.yml

```
@@ -7,6 +7,8 @@ on:
 
 jobs:
   create-release:
+    # This workflow is only of value to PyBaMM and would always fail in forks
```

```suggestion
    # This workflow is only of value to PyBaMM and would always be skipped in forks
```

should we say skipped?